scene/gui/control.cpp

```diff
@@ -56,6 +56,7 @@
 #ifdef TOOLS_ENABLED
 Dictionary Control::_edit_get_state() const {
 	Dictionary s;
+	s["custom_minimum_size"] = get_custom_minimum_size();
```

Comment:
AFAIK no Container can modify minimum size, so why is this here?

Reply:
Yea this probably needs to use regular get_size(), can't remember why i used minimum size, i guess it just so happened to work at time.
Also there could probably be more properties that need to be kept track of that i'm not aware of that could affect the outcome.

Reply:
Anchors and offsets already handle position and size.